WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -764,4 +764,86 @@
       end
     end
   end
+
+  describe 'GET #payment_setup' do
+    context 'when not signed in' do
+      sign_out
+
+      it 'redirects to the sign in page' do
+        get :payment_setup, id: competition, competition: { name: competition.name }
+        expect(response).to redirect_to new_user_session_path
+      end
+    end
+
+    context 'when signed in as an admin' do
+      sign_in { FactoryGirl.create :admin }
+
+      it 'displays payment setup status' do
+        get :payment_setup, id: competition, competition: { name: competition.name }
```

Comment:
What's this `competition: { name: competition.name }` part of the `get ...` all about?